.github/workflows/create-release-on-pr-changes.yml

```diff
@@ -0,0 +1,91 @@
+name: Create release on PR changes
+
+on:
+  pull_request:
+
+jobs:
+  create_release_on_pr_changes:
+    name: "Create release on PR changes"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set release variables
+        id: resolve-release-vars
+        run: |
+          RELEASE_BRANCH=${{ github.event.pull_request.head.ref }}
+          RELEASE_SHA=${{ github.event.pull_request.head.sha }}
+          TAG_NAME=$(echo "branch-${RELEASE_BRANCH}")
+          echo "RELEASE_BRANCH=$RELEASE_BRANCH" >> "$GITHUB_ENV"
+          echo "RELEASE_SHA=$RELEASE_SHA" >> "$GITHUB_ENV"
+          echo "TAG_NAME=$TAG_NAME" >> "$GITHUB_ENV"
+          echo "NPM_VERSION=0.0.0-${TAG_NAME}-${RELEASE_SHA}" >> "$GITHUB_ENV"
+          echo "DIST_FILENAME=dist-$RELEASE_BRANCH-$RELEASE_SHA.zip" >> "$GITHUB_ENV"
```

Comment:
The more I think about it the more I do not agree with adding the SHA to the file name for branches.

As I see it, because of Docker caching we need to make an update to composer to get the latest build of a branch. This will still be the case when adding the SHA to file names - but it will be very obvious because builds will be failing if you do not do so.

On the other hand having a development environment or release branch with a continually updated version will be practically impossible because the SHA for the design system will always be changing. You will have to commit an update the referenced file each time. One developers ability to reinstall dependencies will break because another developer has merged a PR to the design system removing the existing file.

I understand the wish to use a fixed version of these projects but now that we have proper builds on tags I suggest that we use this instead. When a developer wishes to use a static version of the design system - for a demo environment or a proper release - they create a tag. e.g. 2023.50.0-rc1 and reference the files uploaded to that build.

Reply:
Since I have introduced docker [build cache busting](https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/534) it should not be necessary anymore.